tests/lib/cli.js

```diff
@@ -0,0 +1,10 @@
+const { resolve } = require('path');
+// Absolute path to cli file.
+const cliPath = resolve(__dirname, '../../cli.js');
```

Comment:
The original hard-coded `../../../cli.js` path is not compatible with Windows, since that platform uses `\` as a path segment separator. Using `path.resolve` here will result in the correct separator being used for the platform in which this is being run. Additionally, resolving it based on the `__dirname` (which is an absolute path to the directory in which this file exists) removes any uncertainty introduced by relative paths.